src/commands/route/mod.rs

```diff
@@ -0,0 +1,50 @@
+extern crate serde_json;
+
+use cloudflare::endpoints::workers::{DeleteRoute, ListRoutes};
+use cloudflare::framework::apiclient::ApiClient;
+use cloudflare::framework::HttpApiClientConfig;
+
+use crate::http;
+use crate::settings::global_user::GlobalUser;
+use crate::terminal::message;
+
+pub fn list(zone_identifier: String, user: &GlobalUser) -> Result<(), failure::Error> {
+    let client = http::cf_v4_api_client(user, HttpApiClientConfig::default())?;
+
+    let result = client.request(&ListRoutes {
+        zone_identifier: &zone_identifier,
+    });
+
+    match result {
+        Ok(success) => {
+            let routes = success.result;
+            println!("{}", serde_json::to_string(&routes)?);
+        }
+        // TODO: handle route errors with http::format_error
```

Comment:
This will be super nice! Do you plan to get this in for this PR or are you holding off until later?

Reply:
yah that should go in this PR.

Reply:
ah i remember why this was gnarly. i wanted to refactor the logic of extracting an error out of `http::format_error` into its own fn, so that this doesn't break json parsers like jq when it errors.

Reply:
done